core-client/tests/integration_test.rs

```diff
@@ -698,23 +689,23 @@ async fn custodian_backup_recovery<T: DockerComposeContext>(
 #[test_context(DockerComposeCentralizedContext)]
 #[tokio::test]
 #[serial(docker)]
-async fn test_centralized_secure(ctx: &mut DockerComposeCentralizedContext) {
+async fn test_centralized_insecure(ctx: &mut DockerComposeCentralizedContext) {
```

Comment:
Why actually insecure? Isn't it the same for the central case?

Reply:
yes, just called via a different endpoint